### PR TITLE
Huthman King- Unit Testing

### PR DESCRIPTION
**Pull request by kinghuthman**

### CODE REVIEW

File: basic-javascript/project-1.spec.js [hunk 1]
```diff
@@ -1,3 +1,102 @@
...
+    expect(helpers.areSameLength('abcd', 'def')).toBe(false);
+});
+ it('should check if two numbers are the same', () => {
+    expect(helpers.areEqual(8, 9)).toBe(false);
+    expect(helpers.areEqual('8', '8')).toBe(true);
+});
+ it('should check if a number is greater than 50', () => {
+    expect(helpers.greaterThanFifty(49)).toBe(false);
+    expect(helpers.greaterThanFifty(50)).toBe(false);
+    expect(helpers.greaterThanFifty(51)).toBe(true);
```
Comment (bradencodes) [inline basic-javascript/project-1.spec.js:25]: Good way to test edge cases, make the numbers close to the point where the output will change.

File: basic-javascript/project-1.spec.js [hunk 2]
```diff
@@ -1,3 +1,102 @@
...
+});
+ it('should add two numbers and return its value', () => {
+    expect(helpers.add(1,1)).toBe(2);
+    expect(helpers.add(2,2)).toBe(4);
+});
+ it('should subtract two numbers and return its value', () => {
+    expect(helpers.subtract(2,2)).toBe(0);
+    expect(helpers.subtract(4,2)).toBe(2);
```
Comment (bradencodes) [inline basic-javascript/project-1.spec.js:33]: Write tests for when the user inputs something other than numbers.

File: basic-javascript/project-2.spec.js [hunk 3]
```diff
@@ -1,3 +1,76 @@
...
+ it('should add item to the front of an array', () =>{    
+    let arr = [1,2,3];
+    expect(funcs.addItemToFront(arr,1)).toEqual([1,1,2,3]);    
+})
+ it('should return array of words as a sentence', () =>{    
+    expect(funcs.wordsToSentence(['Woah, Woah, Gypsy'])).toBe('Woah, Woah, Gypsy');    
+})
+it('should return whether an array contains an item', () => {
+    expect(funcs.contains([1, 2, 3], 2)).toBe(true);
+    expect(funcs.contains([1, 2, 3], 4)).toBe(false);
```
Comment (bradencodes) [inline basic-javascript/project-2.spec.js:62]: Make what you are testing more clear by using `describe`. The `describe` function will explain what function the tests are running on, and the `it` will describe what the test should return and why. Example: `it('should return true when the array contains the item')` and `it('should return false when the array does not contain the item')`

**Comment by bradencodes** [comment]
## MVP
- [x] Write tests for project-1.js
- [x] Write tests for project-2.js

## Stretch
- [ ] Correct bugs and refactor the code
- [ ] Write tests for arrays.js
- [ ] Write tests for object.js

## What Looks Great
- MVP complete! Nice work
- Descriptive commits
- Code is well formatted and easy to read
- Titles for tests are descriptive, but could be improved a bit

## What Could Be Improved
- Need more commits. You should have one commit for every one change.
- Titles for tests need to be more descriptive
- Tests need to be more thorough and check for edge cases. Also check for failures and data types.

## What Could Be Added
- Add a `describe` for each function you test, and have an `it` for each test.

## Rating (1-3)
**2**
